tests/geocoding_tests.py

```diff
@@ -102,12 +97,13 @@ def test_menu_entry_geocode_exist(self):
     def test_get_columns(self):
         url = "/superset/geocoding/columns"
 
-        table = db.session.query(SqlaTable).all()[0]
+        table = db.session.query(SqlaTable).first()
         table_name = table.table_name
+        columns = reflection.Inspector.from_engine(db.engine).get_columns(table_name)
 
         data = {"tableName": table_name}
         response = self.get_resp(url, json_=data)
-        assert table.columns[0].column_name in response
+        assert columns[0].get("name") in response
```

Comment:
Does this test make sense? We use the same logic in the test (reflection.Inspector) as in the actual code we are testing.

Reply:
jep, because in the method the argument table name will be read from json and in the test method it will be set directliy